src/ui/components/App.tsx

```diff
@@ -81,8 +82,13 @@ function App({ theme, recordingId, modal, updateNarrowMode, setFontLoading }: Ap
   const { claims } = useToken();
   const { availableInvitations } = hooks.useGetAvailableInvitations();
 
-  setTelemetryContext(claims?.hasura.userId, auth.user?.email);
-  // const { loading } = hooks.useMaybeClaimInvite();
+  const userId = claims?.hasura.userId;
+  const userInfo = useGetUserInfo();
+  let isInternal = false;
```

Comment:
Looks like we're not really using `isInternal` anywhere here—are we?

Reply:
Yeah, I meant to set that in the telemetryContext. Will fix. Thanks!